tests/test.sh

```diff
@@ -5,13 +5,20 @@ set -e # Stop on errors
 # Ensure we can activate the environment
 export PATH=$PATH:$HOME/miniconda3/bin
 
+# Set up paths
+ROOT=`pwd`
+
+if [ $# -ne 1 ]; then
+    echo "Write test output to temp file"
```

Comment:
Default condition should be silent, would move the `echo` message to when a custom directory is provided